manifests/rhoai/runtimes/torch_distributed_cuda_28.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: trainer.kubeflow.org/v1alpha1
+kind: ClusterTrainingRuntime
+metadata:
+  name: torch_distributed_cuda_28
```

Comment:
I don't think we can use `_` as it will fail validation and won't be created. 
suggest to make it `-`